tests-integration/src/test_util.rs

```diff
@@ -351,12 +392,33 @@ pub async fn setup_test_prom_app_with_frontend(
     create_test_table(
         frontend.catalog_manager(),
         instance.sql_handler(),
-        ConcreteDataType::timestamp_millisecond_datatype(),
+        ConcreteDataType::timestamp_second_datatype(),
```

Comment:
The root cause is here. You can change it back to `timestamp_millisecond_datatype` and fix the test.